Resources/PO/HomePage.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${USER_NAV_TABS_LOCATOR} =  //ul[@id="myTab"]
+
+
+*** Keywords ***
+Verify User Navigation Tabs Displayed
+    Wait Until Page Contains Element  ${USER_NAV_TABS_LOCATOR}
+
+Verify Home Page loaded
```

Comment:
best way to check that page is loaded - it's verify some unequal element. 
The possible issue that you can catch without this check: 
1) it's a wrong content of the page (e.g content could depends from some conditions: user is logged in/out) 
2) You try manipulate elements that was not be loaded yet